ui-tests/cypress/support/pageObjects/app/Content.js

```diff
@@ -9,6 +9,10 @@ export default class Content extends WebElement {
                .children(this.content);
   }
 
+  getRoot() {
```

Comment:
what is this needed for?
it should not be needed at this level, but you have access to it via parent chain anyway